applications/StructuralMechanicsApplication/custom_elements/shell_thin_element_3D3N.hpp

```diff
@@ -34,6 +34,7 @@ namespace Kratos
 ///@}
 
 class ShellT3_CoordinateTransformation;
+class AdjointFiniteDifferencingBaseElement;

```

Comment:
I think in this case i have to do it like this, because the shell_thin_element_3D3N.hpp is also included in the AdjointFiniteDifferencingBaseElement. So i would create a circular dependency?

Reply:
ok